doc/shell_heavy_guide.md

```diff
@@ -0,0 +1,463 @@
+# 1. Test Objective
+`shel_heavy` test is a kind of the shell test. The test cases that require big disk size or big memory or with heavy workload are categorized as shell_heavy.
+# 2. Execute shell_heavy Test
+To perform the `shell_heavy` test, we need to install CTP first.
+## 2.1 Install CTP
+1. Checkout CTP 
+    ```bash
+    cd ~
+    git clone https://github.com/CUBRID/cubrid-testtools.git
+    cd ~/cubrid-testtools 
+    git checkout develop
+    cp -rf CTP ~/
+    ```
+2. Create and config `CTP/conf/common.conf`   
+    ```
+    git_user=cubridqa
+    git_pwd=PASSWORD
+    git_email=<CUBRIDQA DL email address>
+    default_ssh_pwd=PASSWORD
+    default_ssh_port=22
+
+    grepo_service_url=rmi://192.168.1.91:11099
+
+    qahome_db_driver=cubrid.jdbc.driver.CUBRIDDriver
+    qahome_db_url=jdbc:cubrid:192.168.1.86:33080:qaresu:dba::
+    qahome_db_user=dba
+    qahome_db_pwd=PASSWORD
+
+    qahome_server_host=192.168.1.86
+    qahome_server_port=22
+    qahome_server_user=qahome
+    qahome_server_pwd=PASSWORD
+
+    activemq_user=admin
+    activemq_pwd=PASSWORD
+    activemq_url=failover:tcp://192.168.1.91:61616?wireFormat.maxInactivityDurationInitalDelay=30000
+
+    mail_from_nickname=CUBRIDQA_BJ
+    mail_from_address=<CUBRIDQA DL e-mail address>
+    ```
+3. Add path to `.bash_profile`
+    ```
+    export CTP_HOME=$HOME/CTP
+    export CTP_BRANCH_NAME=develop
+    export CTP_SKIP_UPDATE=0
+    export PATH=$JAVA_HOME/bin:$CTP_HOME/bin:$CTP_HOME/common/script:$PATH
+    ```
+
+## 2.2 Checkout shell_heavy Test Cases   
+The shell_heavy test cases are in the https://github.com/CUBRID/cubrid-testcases-private-ex repository.  
+```
+$ cd ~
+$ git clone https://github.com/CUBRID/cubrid-testcases-private-ex.git
+$ ls ~/cubrid-testcases-private-ex/
+shell  shell_heavy
+```
+```
+$ ls ~/cubrid-testcases-private-ex/shell_heavy/
+_01_cursor_holdability      _06_issues        bug_bts_13188    bug_bts_17504  bug_bts_5359  bug_xdbms48  cbrd_21070  config
+_02_group_and_async_commit  _09_64bit         bug_bts_14738_1  bug_bts_17550  bug_bts_5868  cbrd_20591   cbrd_21119  issue_5765_timezone_support
+_04_misc                    _26_apricot_qa    bug_bts_14738_2  bug_bts_17599  bug_bts_6995  cbrd_21003   cbrd_21186
+_05_addition                _27_aprium_qa     bug_bts_14894    bug_bts_20481  bug_bts_7233  cbrd_21020   cbrd_21361
+_05_multitbl_query_01       _29_features_920  bug_bts_14980    bug_bts_5188   bug_bts_7313  cbrd_21036   cbrd_21362
+_05_multitbl_query_02       bug_bts_10295     bug_bts_17460    bug_bts_5349   bug_bts_8675  cbrd_21049   cbrd_21378
+```
+## 2.3 Quick Start
+1. Install CUBRID   
+    ```
```

Comment:
Add:
"You may install CUBRID as your way or install via a script file `run_cubrid_install` in CTP".

Reply:
OK, I've added it.